### PR TITLE
Cybernetic Organs should now be more worth while

### PR DESCRIPTION
**Pull request by CMOisLing**
Cybernetic organs are shit and really not worth it 90% of the time even for powergamers. So Ive improved them so they should actually be noticable in situations they should be a factor in. Might have buffed the liver too much but I doubt it will be an issue.

-Base level cyber heart now regenerates epinephrine
-Upgraded heart now gives atropine instead
-Base cyber lungs are now old upgraded lungs
-Upgraded lungs are much more robust and should allow you to survive in environments that would harm others
-Base cyber liver is now old upgraded liver
-Upgraded liver should be able to shrug off some fairly bad toxins, and slowly heal toxin damage

#### Changelog

:cl:    
tweak: Cybernetic organs have been improved 
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
I'm for upgraded cyber organs getting buffed but default cyber organs should not have much benefit, if at all; they're supposed to be emergency replacements for when you dont have fresh organs lying around and fuckstick over here had major heart failiure

**Comment by SomeguyManperson** [comment]
Cyber appendix when

**Comment by CMOisLing** [comment]
> Cyber appendix when

I forgot I promised that next update I promise

**Comment by CMOisLing** [comment]
Wait I’m an idiot I forgot to make it printable

File: code/modules/surgery/organs/appendix.dm [hunk 1]
```diff
@@ -42,3 +42,24 @@
...
+	now_failing = "<span class='warning'>NOT AGAIN!</span>"
+	now_fixed = "<span class='info'>My Cyber Appendix is functioning normally.</span>"
+
+/obj/item/organ/appendix/cybernetic/on_life()
+	..()
+	if(inflamed)
+		for(var/datum/disease/appendicitis/A in M.diseases)
+		A.cure()
+		inflamed = FALSE
+		M.emote("chuckle") //you really think that will stop me?
```
Comment (Amelia0010) [inline code/modules/surgery/organs/appendix.dm]: indent these 3 lines

File: code/modules/surgery/organs/appendix.dm [hunk 2]
```diff
@@ -42,3 +42,24 @@
...
+	desc = "One of the most advance cybernetic organs ever created."
+	icon_state = "implant-filter"
+	healing_factor = 3 * STANDARD_ORGAN_HEALING
+	decay_factor = 0.5 * STANDARD_ORGAN_DECAY
+	now_failing = "<span class='warning'>NOT AGAIN!</span>"
+	now_fixed = "<span class='info'>My Cyber Appendix is functioning normally.</span>"
+
+/obj/item/organ/appendix/cybernetic/on_life()
+	..()
+	if(inflamed)
```
Comment (Amelia0010) [inline code/modules/surgery/organs/appendix.dm:57]:
```suggestion
	if(inflamed)
		var/mob/living/carbon/M = owner
```

File: code/modules/surgery/organs/liver.dm [hunk 3]
```diff
@@ -87,18 +87,26 @@
...
-	alcohol_tolerance = 0.001
-	maxHealth = 2 * STANDARD_ORGAN_THRESHOLD //double the health of a normal liver
-	toxTolerance = 15 //can shrug off up to 15u of toxins
-	toxLethality = 0.008 //20% less damage than a normal liver
+	desc = "An upgraded version of the cybernetic liver, designed to handle extreme levels of toxins. It can even heal minor amounts of toxin damage."
+	alcohol_tolerance = 0.0005
+	maxHealth = 1.5 * STANDARD_ORGAN_THRESHOLD //150% health of a normal liver
+	healing_factor = 2 * STANDARD_ORGAN_HEALING //Can regenerate from damage quicker
+	toxTolerance = 20
+	toxLethality = .007
```
Comment (Amelia0010) [inline code/modules/surgery/organs/liver.dm]:
```suggestion
	toxLethality = 0.007
```

File: code/modules/surgery/organs/heart.dm [hunk 4]
```diff
@@ -177,36 +177,37 @@
...
-/obj/item/organ/heart/cybernetic/upgraded
-	name = "upgraded cybernetic heart"
-	desc = "An electronic device designed to mimic the functions of an organic human heart. Also holds an emergency dose of epinephrine, used automatically after facing severe trauma. This upgraded model can regenerate its dose after use."
-	icon_state = "heart-c-u"
-
-/obj/item/organ/heart/cybernetic/upgraded/used_dose()
+/obj/item/organ/heart/cybernetic/upgraded/emp_act()
 	. = ..()
-	addtimer(VARSET_CALLBACK(src, dose_available, TRUE), 5 MINUTES)
+	addtimer(CALLBACK(src, .proc/restart), 80) //Can restart itself after an EMP so it isnt an insta death
```
Comment (Amelia0010) [inline code/modules/surgery/organs/heart.dm]:
```suggestion
	addtimer(CALLBACK(src, .proc/Restart), 80) //Can restart itself after an EMP so it isnt an insta death
```

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

**Review by Amelia0010 — APPROVED** [review]

**Pull request merged**